### ISSUE TITLE
Bone Fracture Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#457]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Bone Fracture Detection</br>
:red_circle: **Aim** : The aim of this project is to detect the bone fractures using deep learning methods.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/pkdarabi/bone-fracture-detection-computer-vision-project</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by pawaspy** [comment]
@abhisheks008 Can you assign me this project under DWOC.
Full name : Pawas Pandey
GitHub Profile Link : github.com/pawaspy
Email ID : pawaspy2633@gmail.com
Participant ID (if applicable): NA
Approach for this Project : I will use sequential model to predict the image and also plot the data which are under right and wrong cattegory.
What is your participant role? (Mention the Open Source program): DWOC.

**Comment by tushtithakur** [comment]
Hi , I'm excited to contribute to this project. Could you please assign me? Looking forward to getting started! @abhisheks008

Full name : Tushti Thakur
GitHub Profile Link : https://github.com/tushtithakur
Email ID : [tushtithakur1234@gmail.com](mailto:tushtithakur1234@gmail.com)
Approach for this Project : Implement different deep learning algorithms using the dataset, evaluate it and compare performance.
What is your participant role? GSSoC 2024

**Comment by Kashish-G** [comment]
Hi, @abhisheks008 I m really excited to contribute to this project. Could you please assign it to me?
Full Name: Kashish Gandhi
GitHub Profile Link: [https://github.com/Kashish-G](url)
Email ID: [kashishgandhi6112003@gmail.com](url)
Approach for this Project: My approach involves leveraging deep learning techniques for bone fracture detection. I plan to explore various convolutional neural network architectures, such as CNNs, and possibly implement more advanced models like U-Net or DenseNet. I aim to preprocess the medical images effectively, train the models on the dataset, and evaluate their performance. Additionally, I intend to employ techniques like data augmentation and transfer learning to enhance the model's accuracy and robustness.
I am excited about the opportunity to contribute to this project and believe that my skills and enthusiasm align well with the goals of the initiative. I am committed to delivering high-quality work and collaborating effectively with the team.
Participant of Gssoc'24

**Comment by abhisheks008** [comment]
Hi @Kashish-G nice approach. But I have assigned you #467, you can work on this issue after completing the assigned one.

**Comment by chirag-garg9** [comment]
I have previous experiences related to such classification projects and will love to solve this issue.

Full name : Chirag Garg

GitHub Profile Link : https://github.com/chirag-garg9
Email ID : [chirag.garg.5293@gmail.com](mailto:chirag.garg.5293@gmail.com)

Participant ID (if applicable): N/A

Approach for this Project :

1.Data augmentation techniques for improving data quality and to make my model more robust.
2.Use SOTA object detection models 
such as YOLO or SSD to get fast and accurate detection and classification scores.
3. Fine-Tuning model the above data and selecting the best performing model using confusion matrix and other useful parameters such as mAp etc.

Participant Role : GSSOC Contributor'24

**Comment by Kashish-G** [comment]
> Hi @Kashish-G nice approach. But I have assigned you #467, you can work on this issue after completing the assigned one.

okay

**Comment by binguliki** [comment]
@abhisheks008 Hey 👋 Bro I have completed My previous Issue Successfully . Can you please assign this task to me I would be using different Pretrained Yolo Models as they would provide better results over the dataset and also fine tune it to get better results

**Comment by abhisheks008** [comment]
I know YOLO will be the go to solution for this project/dataset. Can you implement two more algorithms other than YOLO?

@binguliki

**Comment by binguliki** [comment]
@abhisheks008  I would be using some transfer learning  Faster RCNN and Yolo Algorithms as they would help in detecting low level features also i would also try some data augmentation to as the data available is less so it would help in increasing robustness also i would try tuning different hyperparameters for getting better results.

**Comment by binguliki** [comment]
@abhisheks008  Hey 👋 bro , Can you please assign me the issue I have started working up on it.

**Comment by abhisheks008** [comment]
Hi @binguliki make sure you implement at least 3 architectures for this dataset/project. Issue assigned to you.

**Comment by Pranali3103** [comment]
Full name :Pranali Modi
GitHub Profile Link : https://github.com/Pranali3103
Email ID : [pranali312003@gmail.com]
Approach for this Project : I'll uses different dl algorithms like VGG16, ResNet50, InceptionV3 ,MobileNetV2

What is your participant role? GSSoC 2024

please assign me this issue.

**Comment by abhisheks008** [comment]
Already assigned to other contributor.

**Comment by binguliki** [comment]
Hey 👋 @abhisheks008 , Bro i have some issues and i need your mentorship , i have been working over this issue for past 2 weeks , I have implemented YOLO model for the task . For the other models I didn't find any pretrained models for Faster-RCNN or SSD that i can use for training over the custom dataset . In tensorflow we do have object detection API but it is bit complicated and i could understand.
Can you guide me so that i could the issue.
